docker-compose/docker-compose.yml

```diff
@@ -189,7 +189,7 @@ services:
       - "/bin/bash"
       - "-c"
     command:
-      - "until pg_isready -h compute -p 55433 ; do
+      - "until pg_isready -h compute -p 55433 -U cloud_admin ; do
             echo 'Waiting to start compute...' && sleep 1;
          done"
```

Comment:
I wonder why the existing pipeline doesn't fail using this container https://github.com/neondatabase/neon/actions/runs/5462208177/jobs/9941646873#step:4:250, maybe you have an idea? The changes totally make sense.

Reply:
I think this is because this change doesn’t change the output log of `pg_isready`.
The command `pg_isready` probably does not determine the success/failure of password authentication, but log `accepting connections` as long as a connection is established.
The pipeline is set to succeed if the log has a string of `accepting connections`.
Therefore, both before and after this change, I can see an 'accepting connections' in the log and the pipeline will succeed.

The only contribution of this change is to ensure successful password authentication and eliminate the compute error `FATAL:  password authentication failed for user "root"`.

Reply:
Right, this shouldn't break anything anyway.